### ISSUE TITLE
Refactor form

### ISSUE THREAD

**Issue by manniL** [Developmint/developmint.de#85]

**Comment by johnson-jnr** [comment]
Hi @manniL Can I contribute to this. I would love to
Thanks!

**Comment by manniL** [comment]
Definitely ☺️

**Comment by johnson-jnr** [comment]
Hi @manniL The form we are working on, is it the contact us form (with name, email and inquiry inscriptions)? What do you want to achieve on the form?

Also I noticed something on the contact us form, the page it sit on top (home page) starts to scroll at the end of the form page. It seems that need fixing as well.
Regards!

**Comment by manniL** [comment]
Exactly, it's the form available on https://www.developmint.de/#contact-us (component: https://github.com/Developmint/developmint.de/blob/master/components/ContactUsModal.vue)

> What do you want to achieve on the form?

I've tried to use the new Vue 3 composition API with vuelidate + the form but I wasn't able to find a way around that. This was the actual goal I had in mind.
But any other refactor to make the form less "complex" or "large" is also helpful ☺️


> Also I noticed something on the contact us form, the page it sit on top (home page) starts to scroll at the end of the form page. It seems that need fixing as well.

I'd be quite happy if you could fix that 👍

**Comment by johnson-jnr** [comment]
Hi @manniL I'll see what to do.
